components/pageComponents/standard/Vedlegg/Vedlegg.tsx

```diff
@@ -115,17 +115,19 @@ const Vedlegg = ({ onBackClick, onNext, defaultValues }: Props) => {
             </ReadMore>
           </>
         )}
-      </div>
-      <BodyShort>{formatMessage('søknad.vedlegg.vedleggPåPapir.text')}</BodyShort>
-      <ReadMore
-        header={formatMessage('søknad.vedlegg.vedleggPåPapir.readMore.title')}
-        type={'button'}
-        open={scanningGuideOpen}
-        onClick={scanningGuideOnClick}
-        ref={scanningGuideElement}
-      >
-        <ScanningGuide locale={locale} />
-      </ReadMore>
+      </BodyShort>
+      <BodyShort>
```

Comment:
Liten ting! Tror vi burde være forsiktige med å wrappe komponenter med BodyShort. Da får vi fort sånne <p> inne i <p>-feil. Best å bruke BodyShort kun til ren tekst har jeg tenkt etterhvert. Den kan nok være en div bare her, for alt den gjør er vel å samle komponentene. Enig?

Reply:
fikset i [d66820a](https://github.com/navikt/aap-soknad/pull/52/commits/d66820a68b954bcc8b387b103cae36c42a693790)